exec/vm.go

```diff
@@ -60,6 +61,8 @@ type VM struct {
 	compiledFuncs []compiledFunction
 
 	funcTable [256]func()
+
+	NoPanic bool
```

Comment:
booleans with negation in their name are always a bit harder to grasp.

what about `Recover` or `EnableRecover`?

Also, as it's exported, please document it.